src/reanimated2/PlatformChecker.ts

```diff
@@ -13,7 +13,7 @@ export function isWeb(): boolean {
 }
 
 export function shouldBeUseWeb() {
-  return isJest() || isChromeDebugger() || isWeb();
+  return isJest() || isChromeDebugger() || isWeb() || ((global as any).__DEV__ && process.env.REANIMATED_WEB === 'true');
```

Comment:
I wonder why `isChromeDebugger()` fails?

Reply:
We should probably fix it

Reply:
agree, just not sure how to detect

Reply:
@mvayngrib maybe let's add new function and extract it there? Or check how `isChromeDebugger()` is implemented in Reanimated 3?